src/components/atomic/Loader.vue

```diff
@@ -0,0 +1,33 @@
+<template>
+  <div :class="getClass"></div>
```

Comment:
You might want to add `role="progressbar"` and `aria-label="loading"` :)

Reply:
Now has `aria-label="loading"` (bca12f4). `role="progressbar"` would be misleading because it's not a progress bar — it doesn't have a defined start, end or progress.